source/guides/references/experiments.md

```diff
@@ -17,9 +17,11 @@ Option | Default | Description
 `experimentalComponentTesting` | `false` | Enables component testing using framework-specific adaptors. See {% url "Component Testing" component-testing-introduction %} for more detail.
 `experimentalFetchPolyfill` | `false` | Automatically replaces `window.fetch` with a polyfill that Cypress can spy on and stub. Note: `experimentalFetchPolyfill` has been deprecated in Cypress 6.0.0 and will be removed in a future release. Consider using {% url "`cy.intercept()`" intercept %} to intercept `fetch` requests instead.
 `experimentalRunEvents` | `false` | Allows listening to the {% url "`before:spec`" before-spec-api %} and {% url "`after:spec`" after-spec-api %} events in the plugins file.
+`experimentalSessionSupport` | `false` | Enables support for the Cypress Session API. See {% url `defineSession`  defineSession %} and {% url `useSession`  useSession %} for more detail.
```

Comment:
I can't find the TR PR to comment on, but I recall us before saying that using the word 'support' in an experimental flag is a bit redundant. Could maybe be `experimentalSessions`.